Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -387,17 +462,35 @@ private void SetupActiveEyes()
             if (this.IsStereo)
             {
                 this.LeftEye.stereoTargetEye = StereoTargetEyeMask.Left;
-                this.RightEye.enabled = true;
+                this.LeftEye.enabled = false;

```

Comment:
why do we need these changes?

Reply:
Because we call **Render** manually.

Reply:
I see - can we add a comment indicating that? ie: `// disabling camera as we take control of render order ourselves, see https://docs.unity3d.com/ScriptReference/Camera.Render.html for more info`